dali/test/python/test_utils.py

```diff
@@ -989,3 +992,25 @@ def load_test_operator_plugin():
     except RuntimeError:
         # in conda "libtestoperatorplugin" lands inside lib/ dir
         plugin_manager.load_library("libtestoperatorplugin.so")
+
+
+def is_of_supported(device_id=0):
```

Comment:
Now it's defined twice - the optical flow test (in operator_1) should import it instead of redefining.

Reply:
Fixed